app/assets/javascripts/modules/was_seed_viewer.js

```diff
@@ -1,9 +1,34 @@
+/*global Sly */
 
 (function( global ) {
   'use strict';
   var Module = (function() {
+
     return {
       init: function() {
+        var $wasThumbFrame = $('.sul-embed-was-seed');
+        var dataAttributes = $wasThumbFrame.data();
+
+        // Create the SLY scroll div on fly
+        var $wasThumbScroll = $('<div class="sul-embed-thumb-slider-scroll"><div class="sul-embed-thumb-slider-handle"/></div>');
```

Comment:
The scroll creation and display is tested here https://github.com/sul-dlss/sul-embed/blob/was_view_288/spec/features/was_seed_viewer_spec.rb#L22 
Do you think we need more complex scenarios for testing?